api_poll/api-server-poll.yml

```diff
@@ -42,6 +42,7 @@
                   sudo docker rmi {image_under_test}
                 else
                     echo "Success report scan has been requested for this"
+                    rm -f do_not_poll
```

Comment:
we need to delete the counter as well. I think we need to delete the workspace after success/time out run.

Reply:
@bamachrn : +1
fixed now.